scripts/globals/abyssea/atma.lua

```diff
@@ -432,29 +454,56 @@ end
 xi.atma.onEventFinish = function(player, csid, option, npc)
     local optionSelected = bit.band(option, 0xF)
     if
-        optionSelected == 1 or
-        optionSelected == 4
-    then -- Infuse Atma or Select from history
+        optionSelected == 1
+    then -- Infuse Atma
         local atma = bit.band(bit.rshift(option, 16), 0xFFFF)
+        local orderKeyItem = getSortedKeysFromArray(xi.atma.atmaMods, false)
+        player:messageSpecial(7446, atmaPrice, orderKeyItem[atma])
         addAtma(player, atma)
     elseif optionSelected == 2 then -- Purge atma
         local slot = bit.band(bit.rshift(option, 16), 0xFF)
         if slot == 4 then -- Purge all atma
-            for atmaSlot = 3, 1, -1 do
-                delAtma(player, atmaSlot)
-            end
+            player:messageSpecial(7448)
+            delAllAtma(player)
         else
+            local effect = player:getStatusEffect(xi.effect.ATMA, slot):getPower()
+            player:messageSpecial(7447, effect)
```

Comment:
Use enum here, also, effect should most likely be renamed to effectPower to represent what it is